.github/workflows/docker-develop-image.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted-docker
```

Comment:
@bdevans does this look appropriate? There is a dedicated runner tagged with this label, but are there other changes likely to be needed? I'm not sure we can currently test this workflow without merging.

Reply:
Yes, assuming you have tagged the machine with `self-hosted-docker`(before it was just `docker`). 
Thanks, once that's merged it should start working again...! 🤞

Reply:
Actually, if you tagged it as `docker` then this would mean we could rerun the original job to build the `chaste/release` image and test that it works. The tag could then be changed to the more descriptive `self-hosted-docker` afterwards.